.github/workflows/certora.yml

```diff
@@ -0,0 +1,44 @@
+name: Certora
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+
+      matrix:
+        conf:
+          - Roles
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Install python
+        uses: actions/setup-python@v4
+
+      - name: Install certora
+        run: pip install certora-cli-beta
```

Comment:
It is less stable, but we need it here for compiling MM: otherwise it times out for very simple rules. I don't know exactly what it does, but I'll revert the changes when the fix reaches `certora-cli` (as we have done in morpho-blue)